### PR TITLE
mingw-w64-imagemack: 7.0.1.3 update to latest version

### PR DESCRIPTION
**Pull request by JPeterMugaas**
mingw-w64-imagemack:  Update to altest version

### CODE REVIEW

File: mingw-w64-imagemagick/PKGBUILD [hunk 1]
```diff
@@ -102,6 +108,7 @@ build() {
     --without-ltdl \
     --without-perl \
     --without-x \
+    --without-magick-plus-plus \
```
Comment (Alexpux) [inline mingw-w64-imagemagick/PKGBUILD]: why it here?

File: mingw-w64-imagemagick/PKGBUILD [hunk 2]
```diff
@@ -48,39 +48,44 @@ optdepends=("${MINGW_PACKAGE_PREFIX}-ghostscript: for Ghostscript support"
             )
 options=('staticlibs' 'strip' '!debug' 'libtool')
 source=(https://www.imagemagick.org/download/ImageMagick-${pkgver%.*}-${pkgver##*.}.tar.xz{,.asc}
-        ImageMagick-6.8.8.1-mingw.patch
-        001-relocate.patch
-        002-build-fixes.patch
+        ImageMagick-7.0.1.3-mingw.patch
+        001-7.0.1.3-relocate.patch
```
Comment (Alexpux) [inline mingw-w64-imagemagick/PKGBUILD]: New patches added but seems old not removed from repo

**Comment by svnpenn** [comment]
@JPeterMugaas @Alexpux where is the package for this? I check here:

http://repo.msys2.org/mingw/x86_64

but I only see the old versions

**Comment by Alexpux** [comment]
@svnpenn no package, currently c++ part not building so I skip it. 
Also I'm very busy and will build/upload packages once in a week or two.

**Comment by JPeterMugaas** [comment]
I found a newer 7.0.1.6 release.  I don't think they keep old releases from the 7.0.1 series (they probably should).  But anyway, it seems that bug still is not fixed.

**Pull request merged**